### PR TITLE
BCTHEME-693: Update translation mechanism for config.json values

### PR DESCRIPTION
**Pull request by BC-tymurbiedukhin**
#### What?

Current translation mechanism for page builder values should be replaced with the next one:
Make default config.json values empty and replace it with lang file translations on a template level

#### Tickets / Documentation

[Jira](https://jira.bigcommerce.com/browse/BCTHEME-693)

### CODE REVIEW

File: templates/components/products/card.html [hunk 1]
```diff
@@ -27,22 +25,34 @@
...
-            }}
-        {{else}}
-            {{#or price.sale_price_with_tax.value price.sale_price_without_tax.value}}
+            {{#if theme_settings.pdp_sold_out_label '===' ''}}
+                {{> components/products/product-badge
+                    badge-type='sold-out'
+                    badge_view=theme_settings.product_sold_out_badges
+                    badge_label=(lang "page_builder.pdp_sold_out_label")
+                }}
+            {{else}}
```
Comment (yurytut1993) [inline templates/components/products/card.html:34]: can we replace `if` inside `badge_label attr ` to avoid 2x component usage?

Comment (yurytut1993) [inline templates/components/products/card.html:34]: can we replace` if statement` inside `badge_label attr` to avoid 2x component usage?

Comment (BC-tymurbiedukhin) [reply templates/components/products/card.html:34]: Do you know a way how to implement it?

Comment (yurytut1993) [reply templates/components/products/card.html:34]: Looks like we can only do it inside component. But its up to you

**Review by yurytut1993 — APPROVED** [review]

**Comment by BC-tymurbiedukhin** [comment]
@bc-azvierieva Could you please verify from your side?

**Review by bc-azvierieva — APPROVED** [review]

**Review by bc-alexsaiannyi — APPROVED** [review]

**Pull request merged**